compiler/src/iree/compiler/Codegen/Common/PropagateReshapesByExpansion.cpp

```diff
@@ -326,17 +342,6 @@ void PropagateReshapesByExpansionPass::runOnOperation() {
       };
   linalg::populateFoldReshapeOpsByExpansionPatterns(bubbleExpandShapePatterns,
                                                     bubbleUpExpansionControlFn);
```

Comment:
We can just change the control function to skip fills and use a separate `fill + expand_shape` folder to cleanup such cases.